contributor-docs/adrs/adr-013-primer-experimental-location.md

```diff
@@ -0,0 +1,62 @@
+# ADR 013: Location of experimental react components maintained by Primer
+
+## Status
+
+❌ Not Adopted, `<issue with implementation details goes here>`
+
+&nbsp;
+
+## Context: Experimental components (or drafts) in primer/react
+
+As we work on new components (or rewrites of old components), we often need to start with a "experiment" and not include it in the semantically versioned (semver-ed) main bundle.
+
+[ADR-006](./adr-006-drafts.md) explains our use of "drafts", here are the relevant parts:
+
+- When we are not sure about the API of a component, we put it in a different bundle: "primer/react/drafts". [ADR 010: Merge drafts status into experimental](./adr-010-drafts-are-experimental) recommends changing the name to `experimental`
+- The contract with consumers is that you are opting into a component in which we don't have 100% confidence yet, the API and structure can change.
+
+In [ADR 008: Lowering the barrier to entry](./adr-008-experimental-components) (Sep, 2022), we decided to create room to share experimental components between memex and the monolith with a new repository ([github/experimental-react-components](https://github.com/github/experimental-react-components)). Now that [memex is part of the monolith](https://github.com/github/planning-tracking/issues/1755), that [repository has been archived](https://github.com/github/platform-ux/issues/1248) (Feb, 2023).
```

Comment:
Might want to clarify that `experimental-react-components` is the one that got archived, not `memex` 😅